modules/gitops-release-notes-1-14-3.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-14.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-14-3_{context}"]
+= Release notes for {gitops-title} 1.14.3
+
+{gitops-title} 1.14.3 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, 4.16, and 4.17.
+
+[id="errata-updates-1-14.3_{context}"]
+== Errata updates
+
+[id="RHBA-XXXX:NNNN-gitops-1-14-3-security-update-advisory_{context}"]
+=== RHBA-XXXX:NNNN - {gitops-title} 1.14.3 security update advisory
+
+Issued: 2024-XX-NN
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHBA-XXXX:NNNN[RHBA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
```

Comment:
This is a bit tricky as there is a namespace named `default`. Is this right?
```suggestion
If you have installed the {gitops-title} Operator in the default {gitops-title} namespace, run the following command to view the container images in this release:
```

Reply:
No, we have a different name for the default namespace used for the GitOps operator. So, I would like to politely decline to accept this feedback. I would not like to change what has been followed in the previous z-stream releases.